etl/pipelines/embedding_pipeline.py

```diff
@@ -15,18 +16,26 @@ def _create_embedding_model(self) -> Embeddings:
         """Return an embedding model that will be used to create an embedding store."""
 
     @final
-    def create_embedding_store(self, documents: tuple[Document, ...]) -> VectorStore:
+    def create_embedding_store(
+        self,
+        *,
+        documents: tuple[Document, ...],
+        distance_strategy: DistanceStrategy = DistanceStrategy.EUCLIDEAN_DISTANCE,
+        score_threshold: Annotated[float, Field(json_schema_extra={"min": 0, "max": 1})]
```

Comment:
Nit: not sure why you need the JSON schema stuff here, since we're nowhere near a Pydantic model. In the configurable resource, yeah.

Reply:
I use `json_schema_extra` to add extra properties to tiny types. But it's more appropriate to use Field's `ge` and `le` here, so I have changed this.